R/pipeline.R

```diff
@@ -271,15 +271,15 @@ execute_model_logic <- function(config, input_dir, output_dir) {
   metadata <- list(
     job_id = config@job_id,
     task_id = config@task_id,
-    data_path = empty_str_if_non_existant(config@data@path),
+    data_path = empty_str_if_non_existent(config@data@path),
```

Comment:
What's the diff here? I'm having trouble finding it

Reply:
existent not existant